Bottom: 5a0fcc281f94f30c7502a5b1a6e15d89cd8e5e85
Top:    7ae1fbe3cf953f2bc4fe34ca7036ba66bb9ec43a
Author: Ian Kent <ikent@redhat.com>
Date:   2013-09-23 17:03:14 +0800

Refresh of debug.patch

---

diff --git a/lib/rpc_subs.c b/lib/rpc_subs.c
index 37d7320..46b3e8d 100644
--- a/lib/rpc_subs.c
+++ b/lib/rpc_subs.c
@@ -361,7 +361,7 @@ static unsigned short rpc_getrpcbport(const int proto)
 		"rpcbind", "portmapper", "sunrpc", NULL,
 	};
 	struct servent *entry;
-	struct protent *p_ent;
+	struct protoent *p_ent;
 	unsigned short port;
 	unsigned int i;
